README.md

```diff
@@ -165,7 +165,7 @@ $ gometalinter ./...
 
 # Contributing
 
-We have put all of our contribution guidelines into [CONTRIBUTING.md](https://github.com/prysmaticlabs/geth-sharding/blob/master/sharding/CONTRIBUTING.md)! Check it out to get started.
+We have put all of our contribution guidelines into [CONTRIBUTING.md](https://github.com/prysmaticlabs/prysm/blob/master/sharding/CONTRIBUTING.md)! Check it out to get started.
 
 ![nyancat](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRBSus2ozk_HuGdHMHKWjb1W5CmwwoxmYIjIBmERE1u-WeONpJJXg)
 
```

Comment:
We don't have to mention go-ethereum's licenses at the end, do we?

Reply:
We don't. We aren't modifying go-ethereum - simply importing it, we are gonna be an independent project with GPLv3